build.sbt

```diff
@@ -1,33 +1,32 @@
 import Dependencies._
 
-name := "Tezos-FullStack-Console"
+lazy val root = (project in file(".")).settings(
+  name := "Tezos-FullStack-Console",
+  version := "0.1",
+  scalaVersion := "2.12.8",
+  resolvers += "scalaz-bintray" at "https://dl.bintray.com/scalaz/releases",
+  resolvers += "typesafe" at "http://repo.typesafe.com/typesafe/releases/",
+  resolvers += "Scalac" at "https://raw.githubusercontent.com/ScalaConsultants/mvn-repo/master/",
+
+  libraryDependencies ++= akkaHttp ++ akkaHttpCors ++ akkaStream ++ cats ++ courier ++ greenMail ++ jodaTime ++ logBack ++
+    mySql ++ pureConfig ++ scalactic ++ scalaTest ++ sl4j ++ slick ++ tesozFCTM ++ wireMock ++ testContainers,
+  
+    Defaults.itSettings,
+
+    parallelExecution in IntegrationTest := true,
+    // No need to run tests while building jar
+    test in assembly := {},
+    // Simple and constant jar name
+    assemblyJarName in assembly := s"console.jar",
+    // Merge strategy for assembling conflicts
+    assemblyMergeStrategy in assembly := {
```

Comment:
is that needed?

Reply:
It was like that. will check

Reply:
There are conflicts like that, I would say its a standard practice to define merge strategies like this. Do you have a better idea to handle this ?

Reply:
well, project looks rather simple, no need to bring cannons when you do not need them.

Reply:
no, I think that's fine